packages/docusaurus-theme/src/theme/DocRoot/Layout/Sidebar/index.tsx

```diff
@@ -28,7 +29,7 @@ const getGlobalStyles = ({ euiTheme }: UseEuiTheme) => {
   };
 };
 
-// converted from css moduels to Emotion
+// converted from css models to Emotion
```

Comment:
Nit: Shouldn't that be "modules"?

Reply:
Ha whoops I interpreted that spelling very incorrectly 🙈 
```suggestion
// converted from css modules to Emotion
```